opensearch_py_ml/ml_commons/model_train.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import json
+from typing import Optional
+
+from opensearchpy import OpenSearch
+
+from opensearch_py_ml.ml_commons.ml_common_utils import ML_BASE_URI
+
+
+class ModelTrain:
```

Comment:
One thing just came to my mind is: if we kept adding classes for every single api call in the long run, there'll be a lot of classes. How can we consolidate to reduce number of separate classes?

Reply:
An alternative to this would be to have a single `ModelAPI` class which supports all the api's on the mlcommons. All the support endpoint urls' can be managed in a configuration or as a static variables.

Reply:
if we do want to go with configuration, we can actually configure everything including body in the configuration file and have a single parser which can do API calls. Might minimize the code by a lot.

Reply:
May be we can just create a method like [this](https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_commons/ml_commons_client.py#L567) rather than creating a separate class?

Reply:
I guess I was overthinking. You are right. that actually looks simple enough.

Reply:
We are going to remove this separate Train class then, right?

Reply:
yes, was just running test cases. Will be pushing in another hour